bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -180,11 +182,52 @@ static inline u64 get_on_cpu_cache_miss(u32 *cpu_id)
 	return delta;
 }
 
+static inline void register_new_process_if_not_exist()
+{
+	u64 cgroup_id;
+	u64 pid_tgid = bpf_get_current_pid_tgid();
```

Comment:
Style nit: Split declaration and assignment:

`u64 cgroup_id, pid_tgid, curr_pid, curr_tgid;`